getting-started/run.md

```diff
@@ -8,11 +8,12 @@ Now all we need to do is run the new application:
 1.  **Start the Interbit blockchain node:**
 
     ```sh
+    cd interbit/packages/app-first
     npm run interbit:start
     ```
 
-    > This command continues to run until interrupted. Your application will not
-    > run unless this command is running.
+    > This command continues to run until interrupted. Your application
+    > cannot run unless this command is running.
 
 1.  **Start your application in a new terminal:**
```

Comment:
This is a bit like introducing a new ingredient half way through a recipe. Maybe we should state up front, "Open two terminal sessions. In the first terminal... In the second terminal..."